packages/driver/cypress/e2e/e2e/origin/basic_login.cy.ts

```diff
@@ -1,4 +1,4 @@
-describe('basic login', () => {
+describe('basic login', { browser: '!webkit' }, () => {
```

Comment:
Do we ever want this to run on webkit? Is there a github issue to track if so?

Reply:
Yes, here is the issue for enabling cy.origin for webkit https://github.com/cypress-io/cypress/issues/23799